exceptions.py

```diff
@@ -1,23 +1,74 @@
 # coding: utf-8
+from .resources import Message
 
-class APIException(Exception):
-    pass
+class ResinMissingOption(Exception):
```

Comment:
@dfunckt, I want to use the same class name with NodeJS sdk so our users won't be confused